### ISSUE TITLE
oer2go_cat_url link is broken

### ISSUE THREAD

**Issue by philhb** [iiab/iiab-admin-console#79]
I've been trying to install IIAB and hit an error on the iiab-admin-console section. The error was caused by the script wanting to create a symlink -  TASK [cmdsrv : Create symlink from /common/assets to oer2go_catalog.json]. The file didn't exist. I traced it back to this script and it seems that oer2go_cat_url = 'http://dev.worldpossible.org/cgi/json_api_v1.pl' isn't a valid link. Also dev.worldpossible.org/rachelmods/ isn't valid.

**Comment by tim-moody** [comment]
Yes. I just discovered this today as well. Sometime in the last couple of days the links disappeared. I have contacted OER2GO.

**Comment by tim-moody** [comment]
catalog is back, but not clear of links for downloads work

**Comment by floydianslips** [comment]
The sample buttons for Spanish content don't work. It is a permissions issue with the oer2go server.

**Comment by tim-moody** [comment]
the sample links are provided by oer2go. let them know if they are broken

**Comment by tim-moody** [comment]
the problem of sample links remains as they are all null in the catalog. we would need to scrape or have our own catalog to fix this.
